src/impl_/pyclass.rs

```diff
@@ -847,12 +847,11 @@ pub struct ThreadCheckerImpl<T>(thread::ThreadId, PhantomData<T>);
 
 impl<T> PyClassThreadChecker<T> for ThreadCheckerImpl<T> {
     fn ensure(&self) {
-        if thread::current().id() != self.0 {
-            panic!(
-                "{} is unsendable, but sent to another thread!",
-                std::any::type_name::<T>()
-            );
-        }
+        assert!(
```

Comment:
Wouldn't `assert_eq!` even be nicer as it would include the thread ID in the message?

Reply:
Good idea!